compass/ocean/tests/global_ocean/mesh/so12to30/namelist.init

```diff
@@ -0,0 +1,3 @@
+config_rx1_min_levels = 8
+config_rx1_min_layer_thickness = 2.5
+config_global_ocean_minimum_depth = 20.0
```

Comment:
@cbegeman and I discussed that the 20 m minimum water-column thickness is probably overkill.

What feels more reasonable?  10 m?